gomint-server/src/main/java/io/gomint/server/world/block/Allow.java

```diff
@@ -0,0 +1,27 @@
+package io.gomint.server.world.block;
+
+import io.gomint.server.registry.RegisterInfo;
+import io.gomint.world.block.BlockType;
+
+/**
+ * @author KingAli
+ * @version 1.0
+ */
+@RegisterInfo( sId = "minecraft:allow" )
+public class Allow extends Block implements io.gomint.world.block.BlockAllow{
+
+    @Override
+    public String getBlockId() {
+        return "minecraft:allow";
+    }
+
+    @Override
+    public float getBlastResistance() {
+        return 0; //No Information ???
```

Comment:
I guess allow has a value of 0 since it should behave like air?

Reply:
I think its like Bedrock